en/docs/references/system-requirements.md

```diff
@@ -0,0 +1,81 @@
+# WSO2 Integrator: BI System requirements
+
+Prior to installing WSO2 Integrator: BI, make sure that the appropriate prerequisites are fulfilled.
+
+<table>
+  <tr>
+    <td>
+      <b>Minimum</b>
+      <p>(Suitable for smaller integrations)</p>
+    </td>
+    <td>
+      <ul>
+        <li>
+          0.2 core (compute units with at least 1.0-1.2 GHz Opteron/Xeon processor)
+        </li>
+        <li>
+          512 MB heap size
+        </li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <b>Recommended</b>
+      <p>(Suitable for larger integrations)</p>
+    </td>
+    <td>
+      <ul>
+      <li>
+          1 core (compute units with at least 1.0-1.2 GHz Opteron/Xeon processor)
+        </li>
+        <li>
+          1 GB memory for the container/pod
+        </li>
+      </ul>
+    </td>
+  </tr>
+</table>
+
+## Environment compatibility
+
+- Install a JRE version compatible with the product version listed below, using one of the supported operating systems.
+
+- The details of the tested environments for the WSO2 Integrator: BI are given below.
+
+### Tested operating systems
+
+The WSO2 Integrator: BI runtime is tested with the following operating systems:
+
+| Operating System         | Versions   |
+|--------------------------|------------|
+| Windows                  | 10+       |
+| Ubuntu                   | 24.04      |
+| Red Hat Enterprise Linux | 9   |
+| MacOS                    | 14.6      |
+
+### Tested JREs
+
+The WSO2 Integrator: BI runtime is tested with the following JREs:
+
+| JRE         |Versions|
+|-------------|--------|
+| CorrettoJRE | 21 |
+| AdoptOpenJRE | 21 |
+| OpenJRE     | 21 |
+| Oracle JRE  | 21 |
+
+### Tested DBMSs
+
+The WSO2 Integrator: BI runtime is tested with the following databases:
+
+| DBMS                 | Versions                |
+|----------------------|-------------------------|
+| MySQL                | 8+                      |
+| Oracle               | 12c R2+                 |
+| Microsoft SQL Server | 2005+                   |
+| PostgreSQL           | 8.4+                    |
+
```

Comment:
Why are we listing older versions of DBs. Don't we support much later versions? Btw what is meant by runtime is tested with above dbs. Isn't it the connectors we support/tested. In MI, we have user stores etc, which is required to be tested with DBs, but we don't have such thing with BI, isn't it? @keizer619 / @ThisaruGuruge

Reply:
Lets remove this as we dont have user store